src/server/apis/webhook.ts

```diff
@@ -38,7 +40,83 @@ function getRouter(): express.Router {
     }
   });
 
+  webhookRouter.post('/:action', async (request: express.Request, response: express.Response) => {
+    const loginDetails = await userModel.getLoginFromRequest(request);
+    if (!loginDetails) {
+        response.sendStatus(400);
+        return;
+    }
+
+    if (!request.body.org) {
+      response.status(400).send('No org provided.');
+      return;
+    }
+
+    if (request.params.action === 'add') {
+      try {
+        const githubResponseBody = await github.post(
+          `orgs/${request.body.org}/hooks`, 
+          loginDetails.token,
+          {
+            name: 'web',
+              active: true,
+              events: [
+                '*',
+              ],
+              config: {
+                url: 'https://project-health-internal.googleplex.com/api/webhook',
+                content_type: 'json'
+              }
+          });
+
+        const webhookId = githubResponseBody.id;
+        console.log(`Must add webhook '${webhookId}' to webhook ` +
```

Comment:
Why do we need the webhook id later? I'm not sure if we should be storing state related to the webhooks. To generate the settings page, we should read the orgs & webhooks directly. This is going to be easier than maintaining state since some you can remove it from GitHub settings too.